changes/559.misc.md

```diff
@@ -0,0 +1 @@
+Expressed as an enum so that the exitcode expressed as a number can be explicitly recognized.
```

Comment:
The sentence is not complete. Where is the objective of the verb "expressed"? Also, in changelog we primarily use the imperative form.

Reply:
How about the above comment that was rewritten because others pointed it out?
`To clarify the meaning of the exit code expressed as a magic number, it is expressed as an enum ReturnCode.`

Reply:
I'd suggest:
> Introduce `ExitCode` enum to give concrete semantics to numeric CLI exit codes